CppTicTacToe/tictactoe.cpp

```diff
@@ -13,16 +13,18 @@ using namespace std;
 
 // main global variables and initialization follow here
 
+// Tony Comment
 int round_counter = 1;
 char mark = 'X';
 bool game_over = false;
 int play_again_input = 0;
 int spot_to_fill = 0; // for empty spots, until filled by player and used by player_move() and attempt_to_fill_spot()
+// Tony Comment
 char board_spots[9] = {'1', '2', '3', '4', '5', '6', '7', '8', '9'};
```

Comment:
Instead of using an old school c array, you should use an actual std::array here. It will serve you much better.

Reply:
I updated this just now and pushed it; but I'm not sure why I don't see the new commit on the review-responses branch...

Reply:
Okay, got it now. The new commit shows up on review-responses now.